src/StringUtils.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace LAG\Component\StringUtils;
+
+class StringUtils
+{
+    /**
+     * Camelize a string.
+     *
+     * @param string $id A string to camelize
+     *
+     * @return string The camelized string
+     */
+    public static function camelize($id): string
```

Comment:
There could be a flag to return a UpperCamelCase or lowerCamelCase

Reply:
Done